config/environment.template

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env bash
 set -a
 # hardcoded or derived variables
-ENCRYPTION_KEY=undefined
-export PROJECT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/.. && pwd )"
+export PROJECT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
You'll want to keep the changes in master here on L4-L5

Reply:
will update